### PR TITLE
README: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,12 @@ All `tldr` pages are written in markdown, so they can be edited quite easily and
 pull requests here using Git on the command-line or
 using the GitHub web interface.
 
+But we have already some plugins to different editors to enhance TlDr page editing experience:
```
Comment (owenvoke) [inline README.md:91]:
```suggestion
But we already have some plugins for different editors to enhance the tldr page editing experience:
```

File: README.md [hunk 2]
```diff
@@ -88,6 +88,12 @@ All `tldr` pages are written in markdown, so they can be edited quite easily and
...
+
+- [Emacs](https://github.com/tldr-pages/tldr-emacs-extension.git)
+
+For editors without plugin system we provide set of configs in a [separate repo](https://github.com/tldr-pages/tldr-editor-configs).
```
Comment (owenvoke) [inline README.md:95]:
```suggestion
For editors without a plugin system, we provide a set of configurations in a [separate repo](https://github.com/tldr-pages/tldr-editor-configs).
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
I have a couple of small grammatical changes, but otherwise looks good. 👍🏻

**Review by owenvoke — APPROVED** [review]
❤️

**Review by kbdharun — APPROVED** [review]
LGTM

**Pull request merged**